cmd/juju/cloud/listcredentials.go

```diff
@@ -391,6 +394,10 @@ func formatCredentialsTabular(writer io.Writer, value interface{}) error {
 					credentialNames = append(credentialNames, credentialName)
 				}
 			}
+			if len(credentialNames) == 0 {
+				w.Println(fmt.Sprintf("No credentials for cloud %v to display", cloudName))
```

Comment:
Maybe reads better as
No credentials to display for cloud foo

Reply:
Changed. Thank you.